src/sentry/static/sentry/app/views/organizationDiscover/result/index.jsx

```diff
@@ -101,7 +111,14 @@ export default class Result extends React.Component {
               </a>
             );
           })}
-        </Flex>
+        </ResultViewButtons>
+        <ResultViewDropdownButtons>
+          <ViewsDropdown
```

Comment:
How about extracting the whole VisualizationToggle as a component instead of just the narrow screen version since there seems to be some commonalities between them

Reply:
Sorry, could you clarify?